pandas/core/missing.py

```diff
@@ -645,6 +645,44 @@ def fill_zeros(result, x, y, name, fill):
     return result
 
 
+def mask_zero_div_zero(x, y, result):
```

Comment:
this is not dissimilar from fill_zeros which is not used very much at all. maybe can consolidate.

Reply:
First attempt was to edit fill_zeros to make it work, but I ended up changing it more than I thought appropriate.  I'll take another look.